components/kyma-environment-broker/internal/storage/driver/memory/instance.go

```diff
@@ -262,3 +267,49 @@ func matchFilter(value string, filters []string, match func(string, string) bool
 	}
 	return false
 }
+
+func (s *instances) matchInstanceState(instanceID string, states []dbmodel.InstanceState) bool {
+	if len(states) == 0 {
+		return true
+	}
+	op, err := s.operationsStorage.GetLastOperation(instanceID)
+	if err != nil {
+		// To support instance test cases without any operations
+		return true
+	}
+	fmt.Fprintf(os.Stderr, "%s - %s - %s\n", instanceID, op.Type, op.State)
```

Comment:
please remove this as it's probably for debugging purposes or if it's needed we should use dedicated logger